### PR TITLE
[Leia] cleanups, icon and description add

### PR DESCRIPTION
**Pull request by AlwinEsch**

### CODE REVIEW

**Comment by AlwinEsch** [comment]
After them is in, I create also a Matrix branch of them about the API difference between both

File: appveyor.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 version: BuildNr.{build}
 
-image: Visual Studio 2015
+image: Visual Studio 2017
```
Comment (phunkyfish) [inline appveyor.yml:3]: I though 2015 was for Leia?

File: appveyor.yml [hunk 2]
```diff
@@ -10,14 +10,14 @@ environment:
...
       CONFIG: Release
-    - GENERATOR: "Visual Studio 14 Win64"
+    - GENERATOR: "Visual Studio 15 Win64"
       CONFIG: Release
-    - GENERATOR: "Visual Studio 14 Win64"
+    - GENERATOR: "Visual Studio 15 Win64"
       CONFIG: Release
       WINSTORE: -DCMAKE_SYSTEM_NAME=WindowsStore -DCMAKE_SYSTEM_VERSION="10.0.16299.0"
-    - GENERATOR: "Visual Studio 14 ARM"
+    - GENERATOR: "Visual Studio 15 ARM"
```
Comment (phunkyfish) [inline appveyor.yml:20]: Same VS2015?

Comment (AlwinEsch) [reply appveyor.yml:20]: @Rechi has changed also some other Leia addons to them, only he mean some time ago, the cmake version must stay at 10.0.16299.0

Comment (phunkyfish) [reply appveyor.yml:20]: Ok, cool

**Review by phunkyfish — APPROVED** [review]
LGTM

**Pull request merged**